app/src/main/res/values/strings.xml

```diff
@@ -474,6 +474,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="undo_confirm_negative">Cancel</string>
     <string name="enable_quest_confirmation_title">Enable this quest type?</string>
     <string name="default_disabled_msg_go_inside">This quest type is disabled by default because you will likely have to enter the place to be able to answer a quest of this type.</string>
+    <string name="default_disabled_msg_go_inside_usefullness_warning">This quest type is disabled by default because you will likely have to enter the place to be able to answer a quest of this type. Please, do not enable this quest if answer would be "no" for all objects.</string>
```

Comment:
Typo:

```suggestion
    <string name="default_disabled_msg_go_inside_usefulness_warning">This quest type is disabled by default because you will likely have to enter the place to be able to answer a quest of this type. Please, do not enable this quest if answer would be "no" for all objects.</string>
```